### PR TITLE
Csv

### PR DESCRIPTION
**Pull request by KnazevValeriy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
-.idea/*
```
Comment (old-zoomer) [inline .gitignore]: do not change this file in this pull request

File: .idea/.name [hunk 2]
```diff
@@ -1 +0,0 @@
-InternalProject
```
Comment (old-zoomer) [inline .idea/.name]: do not change .idea folder in this pull request

File: Internal.iml [hunk 3]
```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (old-zoomer) [inline Internal.iml]: do not change .iml files in this pull request

File: allure-results/1a46c3d5-52c2-41ea-8bbe-a6993f3fda60-container.json [hunk 4]
```diff
@@ -1 +0,0 @@
-{"uuid":"1a46c3d5-52c2-41ea-8bbe-a6993f3fda60","name":"com.epam.lab.LogInTest","start":1536460993428,"stop":1536461005377,"children":["ff66f0d8-6c83-4afd-b7a5-173d0e8f2113"]}
```
Comment (old-zoomer) [inline allure-results/1a46c3d5-52c2-41ea-8bbe-a6993f3fda60-container.json]: don't change allure-results folder in this pull request

**Review by MysiukR — APPROVED** [review]

**Review by old-zoomer — CHANGES_REQUESTED** [review]

File: src/test/java/LogInTest.java [hunk 5]
```diff
@@ -1,24 +1,30 @@
...
     public void logInTest() {
-        loginPageBo.logIn("login", "password");
+        loginPageBo.logIn(user.getLogin(), user.getPassword());
     }
 
-    @AfterTest
-    public void quit() {
-        DriverManager.removeDriver();
-    }
+//    @AfterTest
```
Comment (old-zoomer) [inline src/test/java/LogInTest.java:26]: what is this for?

**Review by old-zoomer — APPROVED** [review]

**Review by MysiukR — APPROVED** [review]

**Pull request closed without merging**